### ISSUE TITLE
Clarify behavior when over partition capacity

### ISSUE THREAD

**Issue by bvandersloot-mozilla** [privacycg/CHIPS#85]
Redirecting from [here](https://github.com/privacycg/CHIPS/issues/74#issuecomment-1977237458) as it isn't specific to the 1vs10 KB limit.

I think there is an interesting open question that may be worth aligning on as we have a smaller byte cap. From @edgul, implementing this in Gecko:

> If we assume a per-partition byte limit of 10KB, then it's not hard to imagine the scenario where 10 cookies of 1KB (or perhaps 100 cookies of 100B) are set which stay valid for the foreseeable future and then the next cookie comes along. [...] What is the intended/implemented behaviour? Outright rejection of the newest cookie? Removal of the oldest (still valid) cookie?

cc @DCtheTall

**Comment by johannhof** [comment]
I think this is defined in https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis/#section-5.6-6 (end of "Storage Model" in case the link doesn't work), right? We might want to update the CHIPS spec a bit to amend that section to talk about same domain and same partition-key.

**Comment by bvandersloot-mozilla** [comment]
I agree we should use that method. I just wasn't sure if that was your intent in the explainer!
